README.md

```diff
@@ -1,4 +1,4 @@
-## Amplify for iOS (Developer Preview)
+## Amplify Library for Swift
```

Comment:
"Amplify iOS" is still mentioned a couple of times in this file, is that expected?

Reply:
oh, good catch, will submit a follow-on